TheWarWithin/ShamanRestoration.lua

```diff
@@ -478,7 +487,7 @@ spec:RegisterAbilities( {
         id = 462603,
         known = function() return talent.downpour.enabled and 73920 or nil end,
```

Comment:
I think this can be `73920` w/o a function call.  You can add `talent = "downpour',` below if needed, but I imagine you can't get the `downpour` buff without the talent, so it *shouldn't* be necessary.